healthcheck/client_test.go

```diff
@@ -0,0 +1,59 @@
+package healthcheck
```

Comment:
This one is a matter of preference as I know not everyone agrees with me. However, I like to have my tests in a different package from the code I am testing. This forces you to test via only the public interface of the package and not poke around internally. i.e. consider using the constructor function to instantiate the healthcheck client instead of the struct initialiser. (you will have to inject the http client into the constructor also)